src/app/modules/onboarding/components/wallet-directory-form/wallet-directory-form.component.html

```diff
@@ -1,34 +1,16 @@
-<div *ngIf="loading && !defaultWalletDir">
-  <div class="text-white text-xl mt-4">
-    Loading default wallet directory...
-  </div>
-  <div class="my-4 text-hint text-lg leading-snug">
-    Please wait...
-  </div>
-  <div>
-    <mat-progress-bar mode="indeterminate"></mat-progress-bar>
-  </div>
-</div>
-<div *ngIf="!loading && !defaultWalletDir">
-  Could not retrieve default wallet directory, please try again
-</div>
-<form class="generate-accounts-form" [formGroup]="formGroup" *ngIf="!loading && defaultWalletDir">
+<form class="generate-accounts-form" [formGroup]="formGroup">
   <div class="text-white text-xl mt-4">
     Pick a Wallet Directory
   </div>
   <div class="my-4 text-hint text-lg leading-snug">
-    Enter the directory where you want to store your wallet. By default, a wallet will be created at {{defaultWalletDir}}
+    Enter the directory where you want to store your wallet. By default, a wallet will be created at {{linux}} for Linux machines, {{macos}} for MacOS, or {{windows}} for windows computers, leave blank to use the default value.
```

Comment:
It was on this step that I noticed some UI/UX issues. I felt the path names (ie. `{{linux}}`), when rendered, are difficult to read, especially when three are placed in serial with no intentional line breaks
Couple thoughts I had:
1- Put these three into a stacked `<ul>`
2- Consider adding a reusable css class for `preformatted` text, something like:
```
span.preformatted {
  font-family: monospace;
  white-space: pre;
  border: 1px solid $paper;
  background-color: rgba(0, 0, 0, 0.5);
  display: inline;
  padding-left: 7px;
  padding-right: 7px;
  border-radius: 5px;
}
```
...in `/src/styles.scss` and wrap these in `<span class="preformatted"></span>`... gives it a markdown feel many of us are used to.

Also, it was here that I noticed the wizard is not fitting properly into its container, and the right edge is getting clipped (happens throughout wizard steps)... I can look more into this